tests/test_scene_rendering/test_caching_relalted.py

```diff
@@ -0,0 +1,53 @@
+import os
+import pytest
+import subprocess
+from manim import file_writer_config
+
+from ..utils.commands import capture
+from ..utils.video_tester import *
```

Comment:
It would be nice to get rid of wildcard imports whenever possible.

```suggestion
from ..utils.video_tester import video_comparison
```